dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -20,6 +20,8 @@ default.sslRef=Default SSL repertoire
 default.sslRef.desc=The name of the SSL repertoire that will be used as the default.  The default SSL repertoire called defaultSSLConfig is used if none is specified.
 default.outboundSSLRef=Outbound default SSL repertoire
 default.outboundSSLRef.desc=The name of the SSL repertoire that will be used as the default for outbound SSL connections.  The default SSL repertoire will be used if no SSL repertoire is specified.  This attribute is ignored if the transportSecurity-1.0 feature is not specified.
+default.httpHostNameVerification=Perform hostname verification for http outbound requests.
+default.httpHostNameVerification.desc=Specifies whether SSL outbound hostname verification is enabled. The default is false. If set to true then during the outbound SSL handshake requests the JDK will perform verification that the certificate returned contains a hostname that matches the outbound http request.
```

Comment:
Specifies whether SSL outbound host name verification is enabled. The default is false. If set to true, then during the outbound SSL handshake requests, the JDK will verify that the returned certificate contains a host name that matches the outbound HTTP request.